src/com/google/enterprise/adaptor/prebuilt/DateFilter.java

```diff
@@ -241,7 +254,8 @@ public static DateFilter create(Map<String, String> cfg) {
       format = ISO_8601_FORMAT;
     }
     log.config("format = " + format);
-    SimpleDateFormat dateFormat = new SimpleDateFormat(format);
+    SimpleDateFormat dateFormat = new SimpleDateFormat(
+        "millis".equalsIgnoreCase(format) ? ISO_8601_FORMAT : format);
```

Comment:
If config's format is "millis" then the internal format is "yyyy-MM-dd"? Please add comment explaining.